Oracle.md

```diff
@@ -198,6 +200,31 @@ The TLV serialization of oracle announcements is as follow.
 
 where `signature` is a Schnorr signature over a sha256 hash of the serialized `oracle_event`, using the tag `announcement/v0`.
 
+## Oracle Attestations
+
+After an event occurs, and the oracle creates signatures to attest the outcome, it needs to give them to users.
+An oracle can use an attestation tlv to give users this information.
+
+The TLV serialization of oracle attestations is as follows.
+
+#### Version 0 `oracle_attestation`
+
+1. type: 55400 (`oracle_attestation_v0`)
+2. data:
+    * [`string`:`event_id`]
+    * [`x_point`:`oracle_public_key`]
+    * [`u16`: `nb_attestations`]
```

Comment:
nit: I find it confusing to call these attestations since that's already the name of the enclosing structure. What about just 'signature'? Also it might be ok to have a single `u16` for the number as there should be exactly the same number for each I guess.